CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## XX.XX.XX - 20XX-XX-XX
 
+### PaymentSheet
+* [ADDED][10763](https://github.com/stripe/stripe-android/pull/10763) Added support for updating card details in Link in PaymentSheet.
```

Comment:
This isn't a merchant-facing change, so I wouldn't include it in the changelog.

Reply:
https://github.com/stripe/stripe-android/pull/10763/commits/109e62ff390d70808ab9c676dfcc4ac03f4d4f90